docs/src/app/components/pages/components/Tabs/ExampleLabelWidth.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import {Tabs, Tab} from 'material-ui/Tabs';
+
+const TabsExampleLabelWidth = () => (
+  <Tabs >
+    <Tab labelWidth={100} label="100 width" />
```

Comment:
What's the tradeoff behind explicating the tab width?
We could have make it work without computing the tab width.